src/components/contact/contact-page-form/ContactPageForm.jsx

```diff
@@ -147,7 +147,7 @@ function ContactPageForm() {
             isSubmitButton
             theme="dark"
             widthClass="max-sm:w-full"
-            addedFlexClasses="justify-center"
+            innerOverlayDivClassName="justify-center"
```

Comment:
@LennartWeihs @ndrslmpk 
Das finde ich persönlich echt nicht schön. As discussed before: denke dass muss komplett neu gemacht werden.

Der Button Style sollte auch niemals so variabel sein sondern einheitlich. Z.B. hätte ich justify-center hier nicht erwartet sondern das als "given" gesehen

Reply:
Idealerweise gäbe es denke ich nur einen component für die Gradient-Buttons die dann als children img + text bekommt + className als prop (also anstatt den vier Komponenten die ich da original angelegt hatte).

Reply:
ja @ndrslmpk kannst du nen task erstellen damit das refectored wird?

Reply:
Done.